docs/docs/cmd/spo/folder/folder-retentionlabel-ensure.md

```diff
@@ -0,0 +1,47 @@
+# spo folder retentionlabel ensure
+
+Apply a retention label to a folder
+
+## Usage
+
+```sh
+m365 spo folder retentionlabel ensure [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: URL of the site where the retentionlabel from a file to apply is located
+
+`--folderUrl [folderUrl]`
+: The server-relative URL of the folder that should be labelled. Specify either `folderUrl` or `folderId` but not both.
+
+`i, --folderId [folderId]`
+: The UniqueId (GUID) of the folder that should be labelled. Specify either `folderUrl` or `folderId` but not both.
+
+`--name <name>`
+: Name of the retention label to apply to the folder.
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
+
+Adding a retentionlabel is only supported on subfolders. Adding a retentionlabel to a rootfolder is currently not supported by by this command.
```

Comment:
This is not really clear. A subfolder sounds like a folder within a folder within a library. But what you mean is the rootfolder is the List or Library itself and the subfolder is simply any folder in a library, right? Could you rewrite this?

Reply:
Removed this remark since it is no longer needed because this command can now ensure the retentionlabel to the rootfolder of a document library